torch/_dynamo/variables/tensor.py

```diff
@@ -89,6 +89,16 @@
 )
 
 
+def is_bound_tensor_method(value):
+    return (
+        callable(value)
```

Comment:
Maybe we can check a more specific type for bound tensor method, so that we won't have arbitrary python callable disguised?
```python
>>> type(t.where)
<class 'builtin_function_or_method'>
```

Reply:
Wdym? I think the other checks don't allow arbitrary python callables

Reply:
Yeah, I think we're more relying on the other checks